frontend/src/Root.js

```diff
@@ -16,16 +19,19 @@ const store = createStore(rootReducer,
     : v => v))
 
 sagaMiddleware.run(rootSaga)
+const history = createBrowserHistory()
+history.listen(location => {
+  store.dispatch(RouterActions.Creators.urlChanged(location.pathname))
+})
+store.dispatch(RouterActions.Creators.urlInit(history.location.pathname))
 
 export const Root = () =>
   <Provider store={store}>
     <Theme>
-      <Router>
+      <Router history={history}>
         <Switch>
-          <Route path='/foosball' component={App}/>
-          <Route>
-            <Redirect to='/foosball'/>
-          </Route>
+          <Route path='/:gameName' component={App}/>
```

Comment:
i'd consider renaming App so that the're no confusion between Root and App.
Say `<SpecificGameType>, <GameType>, <Game>` ?

Reply:
Let's call it Game :)

Reply:
fixed in https://github.com/salsita/foosball-rating/pull/141/commits/d00b12891dc1735349c2f42b03bbfc42f1f68aa4